### DISCUSSION TITLE
Is it possible to specify a user-defined path (in URL) for a directory?

### DISCUSSION THREAD

**Discussion by taicaile** [mkdocs/mkdocs#2706] [Q&A]
Hello, I would like to know if there is a way to specify the URL under the directory.

For example, there are two markdown files, `a.md` and `b.md` under directory `docs/public`. The content of the `a.md` is `# a`, and the content  of file `b.md` is `# b`.

By default, the URL for these two files are:

```text
https://example.com/public/a
https://example.com/public/b
```

I expect the URL for these two files are,
```text
https://example.com/p/a
https://example.com/p/b
```

**Comment by timvink** [comment]
That's something you could control outside of `mkdocs`. 

You can `mkdocs build` and copy the output content in `site/` to your server in whatever directory you like, for example `p/`.
